testing/get-lua.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+sudo apt install luarocks
```

Comment:
Sigh... the version of `luarocks` available to apt is 2.4.2 which came out in 2016. I think I have to install `luarocks` from source to get past this current error with the build command.

Reply:
I think if you bump the version of azure-pipeline-templates it'll upgrade to ubuntu 20.04 which should have a new enough version

Reply:
It's a good thought, but 20.04 isn't going to help unfortunately. 2.4.2 is the version that's available in Ubuntu 20.10. 😒 https://packages.ubuntu.com/impish/luarocks